### PR TITLE
Changing SIG to TAG

### PR DESCRIPTION
**Pull request by amye**

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ repository:
   # See https://developer.github.com/v3/repos/#edit for all available settings.
 
   # The name of the repository. Changing this will rename the repository
-  name: sig-security
+  name: tag-security
 
   # A short description of the repository that will show up on GitHub
-  description: 🔐CNCF Special Interest Group on Security -- secure access, policy control, privacy, auditing, explainability and more!
+  description: 🔐CNCF Technical Advisory Group on Security -- secure access, policy control, privacy, auditing, explainability and more!
```
Comment (TheFoxAtWork) [inline .github/settings.yml:8]:
```suggestion
  description: 🔐CNCF Technical Advisory Group on Security (STAG) -- secure access, policy control, privacy, auditing, explainability and more!
```

**Review by TheFoxAtWork — APPROVED** [review]
It is important to note that any links in the repo that are URL not repo path may be broken.  issue #613 identifies this potential break

**Comment by lumjjb** [comment]
It may be sufficient to just change links going forward since github does have an inherent redirection for renaming repos as long as a new repo isn't created in its place (i believe)

**Comment by amye** [comment]
I just tested this with another TAG and it resolved correctly, so I think? we should be ok.

**Review by lumjjb — APPROVED** [review]
lgtm

**Review by pragashj — COMMENTED** [review]
LGTM

**Pull request merged**